server/src/UseCases/Games/GetAllGames/GetAllGamesQuery.cs

```diff
@@ -0,0 +1,7 @@
+﻿using System.Collections.Generic;
+using BGS.ApplicationCore.Entities;
+using MediatR;
+
+namespace BGS.UseCases.Games.GetAllGames;
```

Comment:
also no need to dublicate Games in name of namespace
GetAllGames -> GetAll
CreateGame -> Create